IoTTemperatureTracking/no_hardware.js

```diff
@@ -20,6 +20,7 @@ function generateData(stitchClient){
 }
 
 // Use the API Key to load data
-const stitchClient = Stitch.initializeDefaultAppClient("<STITCH API KEY>");
+const stitchClient = Stitch.initializeDefaultAppClient("<your-app-id>");
+stitchClient.auth.loginWithCredential(new ServerApiKeyCredential("<STITCH API KEY>"));
 generateData(stitchClient);
```

Comment:
you'll want to call this in a then block after the login As it's currently written, `generateData` may be called before the client is authenticated. You'll want something like this:

```javascript
stitchClient.auth.loginWithCredential(new ServerApiKeyCredential("<STITCH API KEY>"))
    .then(() => {
        generateData(stitchClient);
    });
```